eva/executor/load_csv_executor.py

```diff
@@ -0,0 +1,76 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import numpy as np
+import pandas as pd
+
+from eva.configuration.configuration_manager import ConfigurationManager
+from eva.executor.abstract_executor import AbstractExecutor
+from eva.models.storage.batch import Batch
+from eva.planner.load_data_plan import LoadDataPlan
+from eva.storage.storage_engine import StorageEngine
+from eva.readers.csv_reader import CSVReader
+
+
+class LoadCSVExecutor(AbstractExecutor):
+
+    def __init__(self, node: LoadDataPlan):
+        super().__init__(node)
+        config = ConfigurationManager()
+        self.path_prefix = config.get_value('storage', 'path_prefix')
+
+    def validate(self):
+        pass
+
+    def exec(self):
+        """
+        Read the input meta file using pandas and persist data
+        using storage engine
+        """
+
+        # TODO: Should we move this to utils?
+        # utility function to convert bbox to lisst of floats
+        def convert_bbox(bbox):
+            return np.array([np.float32(coord) for coord in bbox.split(",")])
+
+        # Read the CSV file
+        # converters is a dictionary of functions that convert the values
+        # in the column to the desired type
+        # column_list is a list of columns to be read from the CSV file. 
+        # This should match the table schema
+        csv_reader = CSVReader(
+            os.path.join(self.path_prefix, self.node.file_path),
+            converters={"bbox": convert_bbox},
+            column_list=['id', 'frame_id', 'video_id', 'dataset_name', 
```

Comment:
Why the column_list is hard coded?

Reply:
For loading videos, we know the columns are always 'data' and 'id'. But for csv files, it can be anything right? How else can I handle this?

Reply:
I got the problems. Can we get the columns from the csv file? I assume that we need the column_list because we don't want always load all columns. If that's the case, we can have column_list as a file option to the query.